source/features/gha-runs-from-workflow-file.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+const isWorkflowFile = (): boolean => ( ((location.pathname.endsWith('.yaml')) || (location.pathname.endsWith('.yml'))) && location.pathname.split('/', 7)[5] == ".github"  && location.pathname.split('/', 7)[6] == "workflows");
```

Comment:
This check looks very fragile. It should also be moved to https://github.com/sindresorhus/refined-github/blob/master/source/libs/page-detect.ts and tested.

Reply:
Previously I didn't add tests to page-detect when they seemed too specific to be used anywhere else, like `isSingleHTMLFile`.

As for this test, probably something like:
```suggestion
const isWorkflowFile = (): boolean => /\/\.github\/workflows\/.+\.ya?ml$/.test(getRepoPath()!);
```

The problem is that URL-based tests aren't safe in this case (because of branches with slashes), so if you want it to be safe (i.e. DOM-based) it can't be tested `page-detect` either.

Reply:
Should it be made DOM-based and if so what would be a good existing example in the project?